lib/shopify_api/app_server.ex

```diff
@@ -20,13 +20,25 @@ defmodule ShopifyAPI.AppServer do
   @spec set(App.t()) :: :ok
   def set(%App{name: name} = app), do: set(name, app)
 
+  @spec set(App.t(), false) :: :ok
+  def set(%App{name: name} = app, false), do: set(name, app, false)
```

Comment:
🍎 I wonder if this should just be a `set(App.t(), boolean())` with a default of true and killing the ver on line 21.

`def set(%App{name: name} = app, persist \\ true), do: set(name, app, persist)`

Reply:
Like was done in the ShopServer on line 21

Reply:
In this module, there was an existing `set` function that took three args and an existing `set` function that took two, these additional functions were necessary to preserve that existing functionality as the default caused ambiguity that the compiler wasn't happy with. 

It didn't seem like breaking that existing functionality was going to be the right move here...

Reply:
Ah, yeah I wondered down that ally quickly yesterday, I see what you mean. It might be worth while breaking some backwards compatibility here to tidy this up and unify all the funcs across all three genservers, a commit for after this PR though 👍